mparray/special/_special.py

```diff
@@ -31,22 +31,27 @@
 
 
 @vectorize
-def gammainc(x, a):
-    return mp.gammainc(x, a=0, b=a, regularized=True)
+def gammainc(a, x):
+    return mp.gammainc(a, a=0, b=x, regularized=True)
 
 
 @vectorize
-def gammaincc(x, a):
-    return mp.gammainc(x, a=a, b=mp.inf, regularized=True)
+def gammaincc(a, x):
+    return mp.gammainc(a, a=x, b=mp.inf, regularized=True)
```

Comment:
Oops. I was mass-producing these so I suppose I equated `z` with `x`, etc. 

Just to be clear: if the function didn't *work* correctly - if I had the arguments swapped instead of just swapping the names - the tests would have caught it, right?

Reply:
Yeah, I think the tests would have caught it.